app/controllers/application_controller.rb

```diff
@@ -178,6 +178,8 @@ def update_cart # rubocop:disable MethodLength, AbcSize
     # reload appropriate divs / exit
     prepare_catalog_index_vars if params[:controller] == 'catalog'
 
+    # skip renders when called by submit_cart_updates_form
+    return if params[:form]
```

Comment:
I'm not sure if this is the proper way to skip rendering the js when it's being called by the form and instead use the redirect in the submit_cart_updates_form method